src/components/player/index.tsx

```diff
@@ -112,12 +116,24 @@ const VideoResourcePlayer: React.FC<VideoResourcePlayerProps> = ({
           <CurrentTimeDisplay key="current-time-display" order={5} />
           <TimeDivider key="time-divider" order={6} />
           <DurationDisplay key="duration-display" order={7} />
-          <ControlBarDivider key="divider" order={9} className="flex-grow" />
-          <RemainingTimeDisplay key="remaining-time-display" order={10} />
+          <DownloadControl
+            key="download-control"
+            order={8}
+            lesson={videoResource}
+          />
+          <AutoplayControl
+            enabled={true}
+            onDark={true}
```

Comment:
feels like "onDark" is a leftover and this control could be reconsidered in this new position